tests/test_adversary.py

```diff
@@ -17,43 +17,14 @@
 from mart.attack.gradient_modifier import Sign
 
 
-def test_adversary(input_data, target_data, perturbation):
```

Comment:
Why was this test removed?

Reply:
The test covers the case that we run adversary without a model. But the adversary requires a model now.